### PR TITLE
Add translate for Fudy

### PR DESCRIPTION
**Pull request by DongTrieu1998**
The branch is about add translation for Japanese but I make it for Japanese, Chinese, English and Vietnamese.

### CODE REVIEW

**Comment by DongTrieu1998** [comment]
@thanhdat369 please review this MR, thanks

File: fudy/CMakeLists.txt [hunk 1]
```diff
@@ -54,12 +56,16 @@ else()
     if(ANDROID)
```
Comment (ghost) [inline fudy/CMakeLists.txt]: Is it necessary?

File: fudy/NoteScreen/Component/TaskItem.qml [hunk 2]
```diff
@@ -9,11 +9,11 @@ Rectangle {
 
 	property int taskIndex: -1
 	property bool enabled: false
-	property string header
+	property string header: ""
 	property bool noteVisible: false
-	property var noteText
-	property int xaxis
-	property int yaxis
+	property var noteText: ""
```
Comment (ghost) [inline fudy/NoteScreen/Component/TaskItem.qml:14]: string

File: fudy/translations/translation/translation.cpp [hunk 3]
```diff
@@ -0,0 +1,50 @@
+#include "translation.h"
+#include <QtDebug>
```
Comment (ghost) [inline fudy/translations/translation/translation.cpp:2]: remove

File: fudy/translations/translation/translation.h [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+public:
+	explicit Translator(QObject* parent = nullptr);
+
+	Q_INVOKABLE void setLanguage(int lang);
+	QString language() const;
+
+	int vie() const;
+	int eng() const;
+	int chi() const;
+	int jpn() const;
```
Comment (ghost) [inline fudy/translations/translation/translation.h]: please refactor it to locale as vi_VN, en_US, ...

**Review by ghost — APPROVED** [review]
please take a look

File: fudy/NoteScreen/Component/TaskItem.qml (hunk 2 above)
Comment (DongTrieu1998) [reply fudy/NoteScreen/Component/TaskItem.qml:14]: I changed the type of this prop

File: fudy/translations/translation/translation.cpp (hunk 3 above)
Comment (DongTrieu1998) [reply fudy/translations/translation/translation.cpp:2]: Removed

File: fudy/translations/translation/translation.h (hunk 4 above)
Comment (DongTrieu1998) [reply fudy/translations/translation/translation.h]: Names were fixed. Thank you

**Pull request merged**